tools/diagnostics/full_bundle.py

```diff
@@ -30,26 +30,30 @@ def get_dcos_services() -> (bool, str):
         return (True, stdout)
 
 
-def is_service_named(service_name: str, service: dict) -> bool:
+def is_service_named(sdk_service_name: str, dcos_service_name: str) -> bool:
```

Comment:
Have you perhaps investigated why the leading slash is missing in some cases?
Perhaps stripping leading slash from both arguments will make it harder to misuse the function?
Since both arguments are now strings, it could then be called `service_names_match`?

Reply:
Good suggestion 1b334c89b58b85e1eb8e39c276dce1de2eee7ec4